src/buffer.c

```diff
@@ -5278,12 +5278,14 @@ Functions running this hook are, `get-buffer-create',
   Fput (intern_c_string ("erase-buffer"), Qdisabled, Qt);
 }
 
+#ifdef IGNORE_RUST_PORT
```

Comment:
This is never defined. But it makes merges from emacs easier. No more missing functions.